keep/providers/centreon_provider/centreon_provider.py

```
@@ -3,217 +3,234 @@
+                    max_check_attempts=host["max_check_attempts"],
+                    lastReceived=datetime.datetime.fromtimestamp(
+                        host["last_check"]
+                    ).isoformat(),
+                    source=["centreon"],
+                )
+                for host in response.json()
+            ]
+
+        except Exception as e:
+            self.logger.error("Error getting host status from Centreon: %s", e)
+            raise ProviderException(f"Error getting host status from Centreon: {e}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)